tasks/create.yaml

```diff
@@ -29,6 +29,16 @@ tasks:
   - name: k3d-slim-dev-bundle
     description: "Create the slim dev bundle (Base and Identity)"
     actions:
+      - description: "Create base package"
+        task: single-layer
+        with:
+          layer: base
+
+      - description: "Create identity-authorization package"
+        task: single-layer
+        with:
+          layer: identity-authorization
+
```

Comment:
Moved this here rather than keeping the package separate since we have no use-case for the "slim-dev" package to be created on its own outside of the bundle, and if someone wanted to do that they could create the individual layers on their own anyways.